apps/core/management/commands/make_development_fixtures.py

```diff
@@ -34,10 +32,11 @@ def make_development_fixtures(n_pixel_sets=DEFAULT_N_PIXELSETS,
     # PixelSet
     pixel_sets = factories.PixelSetFactory.create_batch(n_pixel_sets)
 
+    index = 0
     for pixel_set in pixel_sets:
```

Comment:
This would have been more pythonic:

```python
for index, pixel_set in enumerate(pixel_sets):
```

Reply:
ah right, I tried without `enumerate()` and it did not work, did not think about this function.